### PR TITLE
add data tracking ids to docs toc nav items

### PR DESCRIPTION
**Pull request by susanev**
fixes https://github.com/pulumi/pulumi-hugo/issues/826

### CODE REVIEW

**Review by cnunciato — APPROVED** [review]
LGTM!

File: themes/default/layouts/partials/docs/toc.html [hunk 1]
```diff
@@ -53,7 +53,7 @@ <h5 class="no-anchor">
             <ul class="{{ $toggle_state }}">
                 <ul class="collapsed">
                     <li class="sidenav-topic {{ $sidenav_selected }}">
-                        <a href="{{ .URL }}">{{ .Name }}</a>
+                        <a href="{{ .URL }}" data-track="toc-{{ .Name | urlize }}">{{ .Name }}</a>
```
Comment (zchase) [inline themes/default/layouts/partials/docs/toc.html:56]:
```suggestion
                        <a href="{{ .URL }}" data-track="toc-{{ .Name | urlize }}-registry">{{ .Name }}</a>
```

File: themes/default/layouts/partials/docs/toc.html [hunk 2]
```diff
@@ -63,7 +63,7 @@ <h5 class="no-anchor">
                 </ul>
                 <ul class="expanded">
                     <li class="sidenav-topic {{ if .HasChildren }} sidenav-parent {{ end }} {{ $sidenav_selected }}">
-                        <a href="{{ .URL }}">{{ .Name }}</a>
+                        <a href="{{ .URL }}" data-track="toc-{{ .Name | urlize }}">{{ .Name }}</a>
```
Comment (zchase) [inline themes/default/layouts/partials/docs/toc.html:66]:
```suggestion
                        <a href="{{ .URL }}" data-track="toc-{{ .Name | urlize }}-registry">{{ .Name }}</a>
```

**Review by zchase — APPROVED** [review]
LGTM but I would suggest adding `-registry` to the ID just to make it a bit clearer in GA what the event is.

**Comment by susanev** [comment]
@zchase but this will add it to all the links in the toc not just the ones that go to registry, i can def still add that though if it would be helpful to you.

**Comment by zchase** [comment]
@susanev Ah just go ahead with how it is now and we can circle back later if we need to clarify more.

**Pull request merged**